__tests__/e2e/synthetics.journey.ts

```diff
@@ -44,15 +44,18 @@ async function goToSyntheticsIntegrationPage(page) {
   if (isUnauthenticated) {
     await logIn(page);
   }
-  await page.waitForTimeout(10 * 1000);
-  await page.click('[data-test-subj="addIntegrationPolicyButton"]');
+  await page.click('[data-test-subj="addIntegrationPolicyButton"]', { timeout: 10 * 1000 });
+  /* We need to ensure that Elastic Synthetics integration page is fully loaded, including the UI logic.
```

Comment:
If you want to ensure the page is fully loaded as a result of the click event
```
const nextPage = await Promise.all([
   await page.click(),
   await page.waitForUrl("") // new page
]);
```

would this help here?